vue-app/src/views/Profile.vue

```diff
@@ -143,9 +176,28 @@ export default class NavBar extends Vue {
       // Log out user
       this.$web3.disconnectWallet()
       this.$store.dispatch(LOGOUT_USER)
-      this.toggleProfile()
+      this.$emit('close')
     }
   }
+
+  private async loadProjects(): Promise<void> {
+    const { recipientRegistryAddress, currentRound, currentUser } =
+      this.$store.state
+    const projects: Project[] = await getProjects(
+      recipientRegistryAddress,
+      currentRound?.startTime.toSeconds(),
+      currentRound?.votingDeadline.toSeconds()
+    )
+    const userProjects: Project[] = projects.filter(({ address }) =>
+      isSameAddress(address, currentUser?.walletAddress)
```

Comment:
So this compares the current user's address to the address the project is using to claim funds. This isn't necessarily the same same address as the one used to submit the application.

Is this what we want? Can we also check against that address used to submit the project? Not sure if we can access the submitters address other than looking through event logs 🤔 

In any case I suppose this is better than nothing 😄

Reply:
We can, would need to update `Project` interface. We look through the event logs to build the list of Projects to begin with...

Currently we call `getProjects` in `recipient-registry-optimistic.ts` which uses our favorite `queryFilter` to filter through and find submission events. These are then run through `decodeProject` and converted into the `Project` type, which is where we lose the `msg.sender` information, which is available as `args._requester`.

I can expand the `Project` interface to also hold a `requester?: string` property, and include this in the check here.

Reply:
Nice!